docs/pai-management/doc/azure/enable-az-rdma.md

```diff
@@ -69,4 +69,32 @@ sudo ./paictl.py utility ssh -p /path/to/cluster/config -f rdma=true -c "sudo /b
 
 ###### ``` 4. Restart all your rdma capable machines in azure portal```
 
-Please communicate with your cluster owner to reboot the rdma machines after the following steps.
\ No newline at end of file
+Please communicate with your cluster owner to reboot the rdma machines after the following steps.
+
+###### ``` 5. Open the switch configuration for az-rdma whose default value is false```
+
+In the [services-configuration.yaml](../../../../examples/cluster-configuration/services-configuration.yaml), please uncomment the file ```cluster.common.az-rdma``` and set its value as ```"true""```.
+
+```YAML
+cluster:
+#
+  common:
+#    clusterid: pai
+#
+#    # HDFS, zookeeper data path on your cluster machine.
+#    data-path: "/datastorage"
+#
+#    # Enable QoS feature or not. Default value is "true"
+#    qos-switch: "true"
+#
+#    # If your cluster is created by Azure and the machine is rdma enabled.
+#    # Set this configuration as  "true", the rdma environment will be set into your container.
+    az-rdma: "true"
+```
```

Comment:
what's the usage of file ```cluster.common.az-rdma``` and set its value as ```"true""```?

Reply:
Grant  the ability of making container rdma capable to cluster.

Reply:
Default value is "false".

Reply:
Here is an example